test/fixtures/full/main.tf

```
@@ -31,35 +31,35 @@ provider "gsuite" {
   version = "~> 0.1.9"
 }
 
-module "vpc" {
-  source          = "terraform-google-modules/network/google"
-  version         = "~> 0.4.0"
-  network_name    = "${var.name}"
-  project_id      = "${var.shared_vpc}"
-  shared_vpc_host = "true"
-
-  subnets = [
-    {
-      subnet_name   = "subnet-01"
-      subnet_ip     = "10.10.10.0/24"
-      subnet_region = "us-east4"
-    },
-  ]
+locals {
+  shared_vpc_subnets = ["projects/${var.shared_vpc}/regions/${google_compute_subnetwork.subnet-01.region}/subnetworks/${google_compute_subnetwork.subnet-01.name}"]
+}
 
-  secondary_ranges = {
-    subnet-01 = [
-      {
-        range_name    = "subnet-01-secondary"
-        ip_cidr_range = "192.168.64.0/24"
-      },
-    ]
-  }
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+  upper   = false
+}
+
+resource "google_compute_network" "network" {
+  name                    = "pf-test-full-${random_string.suffix.result}"
+  routing_mode            = "GLOBAL"
+  auto_create_subnetworks = "true"
```

We shouldn't be automatically creating subnets.
```suggestion
  auto_create_subnetworks = "false"
```